brave/brave-ios
# Fix #2718: Remove sync v1.

## Summary of Changes


This pull request fixes #2718 

## Submitter Checklist:


## Test Plan:

No sync user
- Update from 1.19 to 1.19.1
- Nothing should happen, no alert, sync setting should be gone

Sync user
- Install 1.19 or earlier
- Enable sync chain, can add few bookmarks
- Update to 1.19.1
- Verify that alert message is showing
- Verify that no sync setting is shown, and syncing bookmarks from another device does not work

## Screenshots:



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
